pkg/loop/logger.go

```diff
@@ -52,6 +52,16 @@ func removeArg(args []interface{}, key string) ([]interface{}, string) {
 	return args, ""
 }
 
+// logDebug will parse msg and figure out if it's a panic, this is done here because the hashicorp plugin will push any
+// unrecognizable message from stderr as a debug statement
+func logDebug(msg string, l logger.Logger, args ...interface{}) {
+	if len(msg) > 7 && msg[:6] == "panic:" {
```

Comment:
```suggestion
	if strings.HasPrefix(msg, "panic:") {
```

Reply:
Changed in [`c7fa511` (#384)](https://github.com/smartcontractkit/chainlink-common/pull/384/commits/c7fa511229b14237e6951cb76d428de6db76baf4)